ui/app/components/app/permissions-connect-header/index.scss

```diff
@@ -38,7 +38,14 @@
     color: $Grey-500;
   }
 
+  &__text {
+    width: 100%;
+    text-overflow: ellipsis;
+    overflow: hidden;
+    direction: rtl;
```

Comment:
Hah, I was about to comment on this! You beat me to it.

Reply:
Haha, I was like, "DAMN IT DAMN IT DAMN IT PUSH PUSH PUSH" 😂